src/linear_algebra/affine_space.lean

```diff
@@ -188,37 +198,174 @@ from the given points.  This is intended to be used when the sum of
 the weights is 0; that condition is specified as a hypothesis on those
 lemmas that require it. -/
 def weighted_vsub (w : ι → k) (p : ι → P) : V :=
-weighted_vsub_of_point V w p (classical.choice S.nonempty)
+s.weighted_vsub_of_point V w p (classical.choice S.nonempty)
```

Comment:
Should this definition be turned into an `add_monoid_hom` or maybe even a linear map? Benefit: below you are proving the analogues of `map_zero`, `map_add`, etc... but not `map_sum`, and such. If you turn this into a bundled morphism, it unlocks the corresponding API.

Reply:
The bundled `linear_map` version of `weighted_vsub_of_point` already existed. Changed so that `weighted_vsub_of_point` and `weighted_vsub` are now the bundled versions without having any unbundled versions of those definitions, so that a few more places need to use `weighted_vsub_of_point_apply` but the linear map API is no longer duplicated.

Reply:
Thanks! I think it will pay off in the long run!